### PR TITLE
Cleanup database builder

### PR DESCRIPTION
**Pull request by jacobdekeizer**
- Remove unused import
- Add builder typehint to callbacks
- Fix code style

### CODE REVIEW

File: src/Database/Builder.php [hunk 1]
```diff
@@ -69,30 +68,29 @@ protected function searchWhereInternal($term, $columns, $mode, $boolean)
         }
 
         if ($mode === 'exact') {
-            $this->where(function ($query) use ($columns, $term) {
+            $this->where(function (Builder $query) use ($columns, $term) {
```
Comment (LukeTowers) [inline src/Database/Builder.php]: Is this referring to itself? Why is it required?

Comment (jacobdekeizer) [reply src/Database/Builder.php]: Yes it's referring to itself, because a callback is default mixed typed.

Comment (LukeTowers) [reply src/Database/Builder.php]: Right, but what benefit is this specific change adding?

Comment (jacobdekeizer) [reply src/Database/Builder.php]: Autocompletion for ide's and let the developer know which type is returned

Comment (jacobdekeizer) [reply src/Database/Builder.php:71]: @LukeTowers any update on this?

Comment (LukeTowers) [reply src/Database/Builder.php:71]: LGTM, thanks!

**Pull request merged**